src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -762,10 +821,10 @@ def delete_cc_resource(client, resource_group_name, cluster_name, no_wait):
         utils.arm_exception_handler(e, consts.Delete_ConnectedCluster_Fault_Type, 'Unable to delete connected cluster resource')
 
 
-def update_connectedk8s(cmd, instance, tags=None):
-    with cmd.update_context(instance) as c:
-        c.set_param('tags', tags)
-    return instance
+def update_connectedk8s(client, resource_group_name, cluster_name, tags=None, enable_private_link=None, private_link_scope_resource_id=None, no_wait=False):
```

Comment:
is end user supposed to pass private link details even if he's updating tags? If not, will we pass it as none and reset those values?

Reply:
End user is not supposed to pass private link properties while updating tags(or any other property). And it will not reset the private link behavior